scss/_debug.scss

```diff
@@ -0,0 +1,44 @@
+@if $enable-debugger-classes {
+  .#{$prefix}grid-debug {
+    &::before {
+      position: fixed;
+      top: 0;
+      left: 0;
+      z-index: 2000;
+      padding: $spacer * .5 $spacer;
+      font-family: $font-family-base;
+      color: $black;
+      background-color: $white;
+      border-radius: 0 0 4px; // stylelint-disable-line property-disallowed-list
+      @include font-size($h1-font-size);
```

Comment:
can this get font size from css a variable? then we may able to adjust it like `<body style="--bs-debug-label-font-size: 15px;">`